### ISSUE TITLE
[BUG] medium sized date type inputfield layout error

### ISSUE THREAD

**Issue by tspringborg** [kirbydesign/designsystem#2301]
## Describe the bug
Using a Kirby input field with size type 'md' causes the input text to be displayed a little below the placeholder. The result is something that looks like this.
<img width="682" alt="image" src="https://user-images.githubusercontent.com/1595605/171118962-fc36a044-8e89-45ba-8a5f-97cbd74e3fa6.png">
For comparison, the same markup but with a default-sized input field is displayed below.

## Describe how to reproduce the bug
Simply put in this markup.
```html
<kirby-form-field label="Date input" >
  <input kirby-input type="date" [size]="'md'" />
</kirby-form-field>
```

## Which Kirby version was used?
5.3.1

## Please complete the following information:
- OS: MacOS 12.3.1
- Browser: Chrome 100.0.4896.75
